api/extension-guides/chat.md

```diff
@@ -243,7 +266,9 @@ Users can refer to a chat variable in a prompt by using the `#` symbol. A variab
 
 ![List of variables in chat](images/chat/variables.png)
 
-Variables are resolved independently of the active chat participant. This means that you can use them as a mechanism to share context between different participants. For example, `@workspace` already maintains an index of the current workspace and contributes a variable `#codebase`. Users can include this variable in a prompt to pass the codebase context to another chat participant. It's up to the extension to determine which variables are relevant for processing the user's chat prompt.
+Variables are resolved independently of the active chat participant. This means that you can use them as a mechanism to share context between different participants. For example, `@workspace` already maintains an index of the current workspace and contributes a variable `#codebase`. Users can include this variable in a prompt to pass the codebase context to another chat participant.
+
+Variables and their values are passed as an object bag to the handler in `request.variables`.  The prompt contains varibale references as authored and that it is up to the participant to further modify the prompt, for instance by inlining variable values or creating links to headings which contain the resolved values. Variables are sorted in reverse by their range in the prompt. That means the last variable in the prompt is the first in this list. This simplifies string-manipulation of the prompt.
```

Comment:
```suggestion
Variables and their values are passed as an object bag to the handler in `request.variables`.  The prompt contains variable references as authored and it is up to the participant to further modify the prompt, for instance by inlining variable values or creating links to headings which contain the resolved values. Variables are sorted in reverse order by their appearance in the prompt. That means that the last variable in the prompt is the first in this list. This simplifies string-manipulation of the prompt.
```

Reply:
@isidorn what do you mean by "variable references as authored"? Do you mean, as entered by the user?

Reply:
Yes, as "entered by user". Let me clarify that.